scripts/parse_nagim.py

```diff
@@ -403,21 +692,38 @@ def _add_batch_job(hbatch, cmd, job_name):
 
 
 def _call(cmd):
-    print(cmd)
+    """
+    Call subprocess command locally.
+    """
+    logger.info(cmd)
     subprocess.run(cmd, shell=True, check=True)
 
 
+def _batch_or_call(cmd, hbatch=None, job_name=None):
```

Comment:
I don't really like this, seems a little dangerous to accidentally get it to run. I'd probably say make the user actively decide to _add_batch_job or _call, and have the logic explicitly above.

Reply:
But this function is not user exposed, it's just a helper for this script.

Removed it anyway, calling explicitly `_add_batch_job()` to transfer